velox/exec/TableScan.cpp

```diff
@@ -376,4 +383,12 @@ void TableScan::addDynamicFilter(
   stats_.wlock()->dynamicFilterStats.producerNodeIds.emplace(producer);
 }
 
+void TableScan::setupTracer(const std::string& traceDir) {
```

Comment:
Let's determine this based on the operator type so we don't have to overload the method? Thanks!